### PR TITLE
Fix/update

### PR DESCRIPTION
**Pull request by BoskoGogic95**
Did some small Update functionality (UMS) refactoring.

### CODE REVIEW

File: BE/PoroDev.DatabaseService/Consumers/StorageServiceConsumer/FileReadConsumer.cs [hunk 1]
```diff
@@ -42,5 +45,39 @@ public async Task Consume(ConsumeContext<FileReadModel> context)
...
             await context.RespondAsync<CommunicationModel<FileReadModel>>(response);
         }
+
+        public async Task<FileReadModel> findAdminFiles()
+        {
+            List<FileData> allAdminFiles = (await _unitOfWork.UserFiles.FindAllAsync(userFiles => userFiles.CurrentUser.Email.Contains(""))).ToList<FileData>();
+
+            FileReadModel returnModel = new FileReadModel();
+
+            for (int i = 0; i < allAdminFiles.Count; i++)
```
Comment (petar-kovac) [inline BE/PoroDev.DatabaseService/Consumers/StorageServiceConsumer/FileReadConsumer.cs]: Lets use `allAdminFiles.ForEach(file => { doSomethingHere; }); `

File: BE/PoroDev.DatabaseService/Consumers/StorageServiceConsumer/FileReadConsumer.cs [hunk 2]
```diff
@@ -42,5 +45,39 @@ public async Task Consume(ConsumeContext<FileReadModel> context)
...
+            return returnModel;
+        }
+
+        public async Task<FileReadModel> findUserFiles(Guid userId)
+        {
+            List<FileData> allUserFiles = (await _unitOfWork.UserFiles.FindAllAsync(userFiles => userFiles.CurrentUser.Id.Equals(userId))).ToList<FileData>();
+
+            FileReadModel returnModel = new FileReadModel();
+
+            for(int i = 0; i < allUserFiles.Count; i++)
```
Comment (petar-kovac) [inline BE/PoroDev.DatabaseService/Consumers/StorageServiceConsumer/FileReadConsumer.cs]: Also here, use ForEach()

File: BE/PoroDev.GatewayAPI/Controllers/StorageController.cs [hunk 3]
```diff
@@ -24,7 +24,7 @@ public StorageController(IStorageService storageService, IJwtValidatorService jw
         public async Task<ActionResult<FileUploadRequestGatewayToService>> Upload(IFormFile file)
         {
             Guid userId = await _jwtValidatorService.ValidateRecievedToken(Request.Headers["authorization"]);
-            //Guid userId = Guid.Parse("f7a3a5dd-a69d-4017-b141-a42f1cb2c24d");
+            //Guid userId = Guid.Parse("f8cc4055-75a8-4573-8a7d-5f9d31778559");
```
Comment (petar-kovac) [inline BE/PoroDev.GatewayAPI/Controllers/StorageController.cs]: Remove commented code here

File: BE/PoroDev.GatewayAPI/Controllers/StorageController.cs [hunk 4]
```diff
@@ -33,7 +33,7 @@ public async Task<ActionResult<FileUploadRequestGatewayToService>> Upload(IFormF
         [HttpGet("Download")]
         public async Task<ActionResult<FileDownloadMessage>> Download([FromQuery] string fileId)
         {
-            //Guid userId = Guid.Parse("f7a3a5dd-a69d-4017-b141-a42f1cb2c24d");
+           // Guid userId = Guid.Parse("f8cc4055-75a8-4573-8a7d-5f9d31778559");
```
Comment (petar-kovac) [inline BE/PoroDev.GatewayAPI/Controllers/StorageController.cs]: Remove commented code here

**Review by damjan21 — APPROVED** [review]

**Pull request merged**